.github/workflows/codeql-analysis-cron-schedule.yml

```diff
@@ -0,0 +1,69 @@
+# This is an alternative to the codeql-analysis.yml that only contains a scheduled evaluation of CodeQL
+# The action runs for all branches defined in jobs.analyze.strategy.matrix.branches.
+# Every new production branch (minor release branches) should be added to this list.
+
+name: "Scheduled CodeQL"
+
+# run twice a week at a random time on Sunday and Wednesday evening so its available the next morning
+on:
+  schedule:
+    - cron: '42 21 * * 0,3'
+
+jobs:
+  analyze:
+    name: Analyze
+    runs-on: ubuntu-latest
+    permissions:
+      actions: read
+      contents: read
+      security-events: write
+
+    strategy:
+      fail-fast: false
+      matrix:
+        # CodeQL runs on these branches
+        branches:
+          - 'master'
+          - 'V5.4'
+          - 'V6.0'
```

Comment:
Why not "V6*"? And hopefully the future us don't forget to add V7... (THey will)

Reply:
pattern matching doesn't work here

Reply:
so yes, this will have to be added to every future minor/major release...

Reply:
see #3522